src/Map.fs

```diff
@@ -11,50 +11,101 @@ type IMap =
 
 module Map =
     let WidgetKey = Widgets.register<Map> ()
-    
+
+    let MapType =
+        Attributes.defineEnum<MapType> "Map_MapType" (fun _ newValueOpt node ->
+            let map = node.Target :?> Map
+
+            let value =
+                match newValueOpt with
+                | ValueNone -> MapType.Street
+                | ValueSome v -> v
+
+            map.MapType <- value)
+
+    let IsShowingUser = Attributes.defineBindableBool Map.IsShowingUserProperty
+
+    let TrafficEnabled = Attributes.defineBindableBool Map.TrafficEnabledProperty
+
     let HasScrollEnabled = Attributes.defineBindableBool Map.HasScrollEnabledProperty
+
     let HasZoomEnabled = Attributes.defineBindableBool Map.HasZoomEnabledProperty
-    let IsShowingUse = Attributes.defineBindableBool Map.IsShowingUserProperty
-    
-    let TrafficEnabledProperty = Attributes.defineBindableBool Map.TrafficEnabledProperty
-    
-    let MoveToLastRegionOnLayoutChange = Attributes.defineBindableBool Map.MoveToLastRegionOnLayoutChangeProperty
-    
-    let MapType =
-        Attributes.defineEnum<MapType>
-            "Map_MapType"
-            (fun _ newValueOpt node ->
-                let map = node.Target :?> Map
-
-                let value =
-                    match newValueOpt with
-                    | ValueNone -> MapType.Street
-                    | ValueSome v -> v
-
-                map.MapType <- value)
-            
-    let ItemsSource<'T> =
-        Attributes.defineBindable<WidgetItems<'T>, System.Collections.Generic.IEnumerable<Widget>>
-            Map.ItemsSourceProperty
-            (fun modelValue ->
-                seq {
-                    for x in modelValue.OriginalItems do
-                        modelValue.Template x
-                })
-            (fun a b -> ScalarAttributeComparers.equalityCompare a.OriginalItems b.OriginalItems)
 
+    let MoveToLastRegionOnLayoutChange =
+        Attributes.defineBindableBool Map.MoveToLastRegionOnLayoutChangeProperty
+
+    let RequestedRegion =
+        Attributes.defineSimpleScalarWithEquality<MapSpan> "Map_RequestedRegion" (fun _ newValueOpt node ->
+            let map = node.Target :?> Map
+
+            match newValueOpt with
+            | ValueNone -> ()
+            | ValueSome mapSpan -> map.MoveToRegion(mapSpan))
```

Comment:
Maybe it would be better to check against `oldValueOpt` to check if the region is different.

Reply:
Humm let me check what I can do here :)

Reply:
Forgot that `Attributes.defineSimpleScalarWithEquality` was already doing the check. All good then :)